src/main/java/nus/climods/logic/parser/ExitCommandParser.java

```diff
@@ -0,0 +1,29 @@
+package nus.climods.logic.parser;
+
+import static nus.climods.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import nus.climods.logic.commands.ExitCommand;
+import nus.climods.logic.parser.exceptions.ParseException;
+
+/**
+ * Parser that parses the exit command from user.
+ */
+public class ExitCommandParser implements Parser<ExitCommand> {
+
+    private static final String DEFAULT_ERROR_MESSAGE =
+            String.format(MESSAGE_INVALID_COMMAND_FORMAT, ExitCommand.MESSAGE_USAGE);
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddCommand and returns an AddCommand object
+     * for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ExitCommand parse(String args) throws ParseException {
+        String trimmedInput = args.trim();
+        if (trimmedInput.isEmpty()) {
```

Comment:
Actually is there a need to check args here? In CliModsParser, isn't commandWord is passed as a parameter which must be equal to ExitCommand.COMMAND_WORD for this to have run?

Reply:
args is the arguments after `exit`
but i just realised that the user input is trimmed alr making the check duplicated

Should we remove parser for redundant classes or just leave it in?

Reply:
keep for better DX? if not its the only func w/o parser